configure.ac

```diff
@@ -63,7 +70,7 @@ esac
 
 # set hardening flags
 
-LDFLAGS="$LDFLAGS -Wl,-z,relro,-z,now"
+LDFLAGS="-Wl,-z,relro,-z,now"
```

Comment:
Don't we need to maintain whatever `LDFLAGS` the user passes in?

Reply:
Good catch, fixed !